cmd/panfs.go

```diff
@@ -1911,7 +1910,8 @@ func (fs *PANFSObjects) isConfigAgentObject(bucket, object string) bool {
 
 	switch {
 	case strings.HasPrefix(object, "buckets/"):
```

Comment:
maybe we can be more exact here and just check regexp (I may put wrong metadata file name from my memory btw):

```
buckets/[^/]+/.metadata.bin
```